elements/custom-toolbar.html

```diff
@@ -0,0 +1,13 @@
+<dom-module id="custom-toolbar" extends="paper-toolbar">
+  <template>
+    <paper-toolbar class="paper-header">
+      <h1>CUSTOM</h1>
+    </paper-toolbar>
+  </template>
+  <script>
+  Polymer({
+    is: 'custom-toolbar',
+    extends: 'paper-toolbar'
+  });
+  </script>
+</dom-module>
```

Comment:
You may want to create a folder named fixture inside test and move this file.  Also, I'm not sure why you want to extend from `paper-toolbar`.